src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocsummarylocation/InputJavadocSummaryLocationTrailingSpace.java

```diff
@@ -0,0 +1,23 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocsummarylocation;
+
+/**
+ * config: default
+ */
+public final class InputJavadocSummaryLocationTrailingSpace {
+
+    /**
+     * Trailing space // OK
+     */
+    private int space;
+
+    /**********************
+     * Trailing asterisks * // OK
+     **********************/
+    private int asterisks;
+
+    /*** *
+     * Asterisk, then space, then another asterisk. // violation
```

Comment:
please add:
```
    /**
     * there is space in first line, weird corner case, we treat it as // violation
     * to avoid violation space need to removed.
```

Reply:
Is this addressed ?

Please reply done to items you addressed

Reply:
done.